common/src/main/java/com/google/auto/common/MoreElements.java

```diff
@@ -189,5 +197,41 @@ public static boolean isAnnotationPresent(Element element,
     return Optional.absent();
   }
 
+  /**
+   * Returns a {@link Predicate} that can be used to filter elements by {@link Modifier}.
+   * The predicate returns {@code true} if the input {@link Element} has all of the given
+   * {@code modifiers}, perhaps in addition to others.
+   * <p>
+   * Here is an example how one could get a List of static methods of a class:
```

Comment:
Nit: Google style puts <p> at the beginning of the line, not on a separate line:
```
 *
 * <p>Here is an example...
```

Same thing in the other method.